subsys/bluetooth/adv_prov/providers/fast_pair.c

```diff
@@ -82,7 +82,7 @@ static int get_data(struct bt_data *ad, const struct bt_le_adv_prov_adv_state *s
 		    struct bt_le_adv_prov_feedback *fb)
 {
 	static uint8_t buf[ADV_DATA_BUF_SIZE];
-	struct bt_fast_pair_adv_config adv_config;
+	struct bt_fast_pair_adv_config adv_config = { 0 };
```

Comment:
All of the fields are filled a few lines below before the structure is used. Is the structure actually uninitialized?

Reply:
Discussed offline. There is no bug in the current code. Still, zeroing struct fields would not hurt here.

Reply:
Maybe. But it is good practice to clean-up local structure when field filling is spread across whole function body